internal/charmstore/migrations_integration_test.go

```diff
@@ -348,7 +348,7 @@ var migrationFromDumpBaseEntityTests = []struct {
 				Read:  []string{"charmers"},
```

Comment:
I think we want another integration test in here please, to test the new migration, because AFAIK the current charm store is deployed at 4.5.6, so we want a test that migration from that version works OK.